src/components/text-editor/__internal__/toolbar/toolbar-button/toolbar-button.spec.js

```diff
@@ -80,36 +81,4 @@ describe("ToolbarButton", () => {
       );
     });
   });
-
-  describe("Tooltip", () => {
-    const ToolBarButtonWithToolTip = (props) => {
-      const defaultProps = {
-        onKeyDown,
-        onMouseDown,
-        onMouseOver,
-        onMouseLeave,
-        tooltipMessage: "foo",
-        tooltipPosition: "top",
-        tooltipAlign: "center",
-        tooltipVisible: false,
-      };
-      return (
-        <ToolbarButton {...defaultProps} {...props}>
-          foo
-        </ToolbarButton>
-      );
-    };
-
-    it("is displayed when `tooltipVisible` is true", () => {
```

Comment:
Are these not still valid tests? `tooltipVisible` is still a prop of `ToolbarButton` from looking above

Reply:
Yeah same as above forgot to add back in, thanks will do when the reviews are complete

Reply:
Ah actually I moved these tests into the Toolbar as it was where the buttons are wrapped in the `Tooltip`